src/pages/components/tag/style.mdx

```diff
@@ -56,12 +68,16 @@ with no spaces.
 All tags have the same height. However, the width of tags varies based on the
 amount of content. All four corners of a tag are rounded with a 24px radius.
 
-| Element | Property                    | px / rem | Spacing token |
-| ------- | --------------------------- | -------- | ------------- |
-| Tag     | height                      | 24 / 1.5 | –             |
-|         | radius                      | 24px     | –             |
-|         | margin                      | 8 / 0.5  | `$spacing-03` |
-|         | padding-left, padding-right | 8 / 0.5  | `$spacing-03` |
+| Element   | Property                    | px / rem   | Spacing token |
+| --------- | --------------------------- | ---------- | ------------- |
+| Big tag   | height                      | 24 / 1.5   | –             |
```

Comment:
Same as the other comment, can this be `Medium tag`? for consistency

Reply:
Ok updated